qa/setup_packages.py

```diff
@@ -453,7 +453,14 @@ def get_pyvers_name(self, url, cuda_version):
                         { "100" : [
                             PckgVer("2.2.0", dependencies=["protobuf<4"])],
                           "110" : [
-                            PckgVer("2.2.0", dependencies=["protobuf<4"])]})
+                            PckgVer("2.2.0", dependencies=["protobuf<4"])]}),
+                CudaPackage("numba",
+                        { "100" : [
+                            PckgVer("0.55.2", python_min_ver="3.7"),
+                            PckgVer("0.53.1")],
```

Comment:
```suggestion
                            PckgVer("0.53.1",  python_max_ver="3.6")],
```

Reply:
Otherwise we will test  0.55.2 and 0.53.1 for pythons 3.7 and above.

Reply:
Done. I had that and removed it before :)